doc/api/fs.md

```diff
@@ -33,8 +37,11 @@ fs.unlink('/tmp/hello', (err) => {
 });
 ```
 
-Exceptions that occur using synchronous operations are thrown immediately and
-may be handled using `try…catch`, or may be allowed to bubble up.
+## Synchronous Example
+
+The synchronous form blocks the Node.js event loop and further JavaScript
+execution until the operation is complete. Exceptions are thrown immediately
+and may be handled using `try…catch`, or may be allowed to bubble up.
```

Comment:
nit: ... and it can be caught using `try..catch` else it will be bubble up.

Reply:
```suggestion
and can be handled using `try…catch`, or can be allowed to bubble up.
```

Reply:
> nit: ... and it can be caught using `try..catch` else it will be bubble up.

If we do something like that, I'd suggest splitting it into two sentences. The second sentence would be something like "If `try..catch` is not used, the exception bulbbles up."